Source/Flow/Public/Interfaces/FlowNodeWithExternalDataPinSupplierInterface.h

```diff
@@ -0,0 +1,34 @@
+// Copyright https://github.com/MothCocoon/FlowGraph/graphs/contributors
+
+#pragma once
+
+#include "UObject/Interface.h"
+#include "FlowNodeWithExternalDataPinSupplierInterface.generated.h"
+
+class IFlowDataPinValueSupplierInterface;
+struct FFlowPin;
+
+// Interface for special flow node types that support an external data pin supplier.
+// The primary (only?) implementing node is UFlowNode_Start, which supplies its pin data externally from
+// either the SubGraph that instanced the graph that is being started.
+UINTERFACE(MinimalAPI, NotBlueprintable, DisplayName = "Flow Node With External Data Pin Value Supplier Interface")
+class UFlowNodeWithExternalDataPinSupplierInterface : public UInterface
+{
+	GENERATED_BODY()
+};
+
+class FLOW_API IFlowNodeWithExternalDataPinSupplierInterface
+{
+	GENERATED_BODY()
+
+public:
+
+	// Set the external DataPinValueSupplier for this node to use.
```

Comment:
Special interface for flow nodes that source their pin data from an external source.  The only one right now is Start Node.

I did have CustomInput also be able to specify pins and source them externally, but I removed that support (for now?) because it was confusing on the SubGraph inputs, which ones went with the Start node and which pins went with the CustomInput(s).  

I felt the feature was tighter and cleaner by limiting it to just the Start node, but I expect there may be other nodes in the future that can source a external pin supplier.